examples/hello_rust/hello_rust_main.rs

```diff
@@ -21,54 +21,115 @@
 /****************************************************************************
  * Attributes
  ****************************************************************************/
+ 
+ #![no_main]
+ #![no_std]
+ 
+ /****************************************************************************
+  * Uses
+  ****************************************************************************/
+ 
+ use core::panic::PanicInfo;
+ use core::ffi::{ c_char, c_int, c_void };
+ 
+ /****************************************************************************
+  * Externs
+  ****************************************************************************/
+ 
+ extern "C" {
+     pub fn printf(format: *const u8, ...) -> i32;
+     pub fn puts(s: *const c_char) -> c_int;
```

Comment:
```suggestion
     pub fn puts(s: *const u8) -> i32;
```

Let's standardise: c_char becomes u8, c_int becomes i32

Reply:
Okay,